src/lib/utils.ts

```diff
@@ -163,7 +166,13 @@ export const formatTimestamp = (
 // Check whether the input is a Radicle ID.
```

Comment:
There's no such thing as a "radicle id" anymore in heartwood. There are node ids (or NIDs), which are public keys, and repository ids (or RIDs) which are Oids.

Reply:
The former is 32 bytes long, and the latter is 20 bytes long. Both use a multi-base encoding using the bitcoin base58 encoding, though RIDs are often displayed as URNs, with the `rad:` prefix.

Reply:
Ok this solves my other response wrt to a spec. So we need to two format functions, one for Oids with or without the prefix and one for public keys